AppServices/Mappers/CustomerBankInfo/Requests/UpdateCustomerBankInfoRequest.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AppServices.Mappers.CustomerBankInfo.Requests
+{
+    public class UpdateCustomerBankInfoRequest
+    {
+        public AccountRequestType RequestType { get; set; }
+        public decimal Value { get; set; }
+    }
+
+    public enum AccountRequestType
+    {
+        [Description("Depositar")]
+        Deposit = 1,
+        [Description("Sacar")]
+        Withdraw = 2
+    }
```

Comment:
```suggestion
    public enum AccountRequestType
    {
        [Description("Depositar")]
        Deposit = 1,
        [Description("Sacar")]
        Withdraw = 2
    }
```
Eu sugiro que crie esse Enum separado em outra classe